<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadWidget</class>
 <widget class="QWidget" name="DownloadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>112</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>112</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QToolButton" name="playBtn">
     <property name="styleSheet">
      <string notr="true">QToolButton{
   background-color: rgba(0, 82, 112, 100);
    color: white;
    border: 1px;
}

QToolButton:hover{
    background-color: rgba(0, 82, 112, 200);
    color: white;
    border: 1px;
}

QToolButton:disabled{
    background-color: transparent;
    color: gray;
}</string>
     </property>
     <property name="text">
      <string>播放</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>399</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="pauseBtn">
     <property name="styleSheet">
      <string notr="true">QToolButton{
   background-color: rgba(0, 82, 112, 100);
    color: white;
    border: 1px;
}

QToolButton:hover{
    background-color: rgba(0, 82, 112, 200);
    color: white;
    border: 1px;
}

QToolButton:disabled{
    background-color: transparent;
    color: gray;
}</string>
     </property>
     <property name="text">
      <string>暂停</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="fileNameLbl">
     <property name="text">
      <string>none</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="startBtn">
     <property name="styleSheet">
      <string notr="true">QToolButton{
   background-color: rgba(0, 82, 112, 100);
    color: white;
    border: 1px;
}

QToolButton:hover{
    background-color: rgba(0, 82, 112, 200);
    color: white;
    border: 1px;
}

QToolButton:disabled{
    background-color: transparent;
    color: gray;
}</string>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="deleteBtn">
     <property name="styleSheet">
      <string notr="true">QToolButton{
   background-color: rgba(255, 255, 255, 0);
    color: white;
    border: 1px;
}

QToolButton:hover{
    background-color: rgba(255, 255, 255, 10);
    border: 1px;
}

QToolButton:pressed{
    background-color: rgba(255, 255, 255, 50);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="speedLbl">
     <property name="text">
      <string>0Kb/s</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
